share/translations/keepassx_ar.ts

```diff
@@ -4094,5 +4094,9 @@ Please unlock the selected database or choose another one which is unlocked.</so
         <source>Parent window handle</source>
         <translation>زر النافذة الأم</translation>
     </message>
+    <message>
```

Comment:
> I've also updated the translation files and I'd like a thumbs up on that.

Please revert that. We will update translations before the release. There is no point in pulling them in now. They just convolute the PR.